rasa/utils/tensorflow/converter.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
```

Comment:
Should this file really be in `rasa.utils.tensorflow`? Does it have any specific relation with TF?

Reply:
Oh my god, definitely no! Supposed to be one level up, thanks for spotting that!